### DISCUSSION TITLE
Assigning subtasks to a team member

### DISCUSSION THREAD

**Discussion by frankulatowski-scm** [Leantime/leantime#611] [Ideas]
We put stories from the project backlog into the sprint backlog and split them into smaller subtasks there. Unfortunately, it is not possible to assign a subtask to a team member. This would be useful because the team member could see these tasks on their dashboard.

Is this a feature that is still missing and will be added at some point? Or is the concept of Leantime different in this point and we should not proceed as described above?

Regards, Frank

**Comment by marcelfolaron** [comment]
Hi,

Thanks for the feedback. Given the direction and usage within the community
I believe we should make it possible to assign subtasks to other users.

For that to work, we will have to make sure they appear in Kannan boards
and lists though.

**Comment by frankulatowski-scm** [comment]
Hi,

Thanks for your response. 

Another idea in this context: It is possible to create a new, separate task from a subtask with a mouse click (with transfer of all entries). The subtask is removed from the original story.

I would then first move a story into a sprint and create new tasks from the subtasks there. I can then assign these new tasks to team members. This works similar to Trello, where I can create new cards from an element of a checklist. 

Thanks for your efforts and your work on the really great Leantime app

Regards, Frank

**Reply by marcelfolaron** [reply]
That should be fairly simple. So basically you would change the type from subtask to Task and remove the relationship between the parent and child?

**Comment by frankulatowski-scm** [comment]
Yes, that's exactly how it works in Trello or Microsoft Planner. There you can create new cards from a checklist (there are no sub-tasks).

However, I would prefer to be able to edit the tasks in detail directly in the user story card of Leantime. But currently, the editing of the sub-tasks is still very limited. 

Probably the other suggestion is easier to implement though - that's my guess as a non-programmer :-)
